server/app/routes/v1/grids/grid_certificates.rb

```diff
@@ -0,0 +1,20 @@
+V1::GridsApi.route('grid_certificates') do |r|
+
+  # POST /v1/grids/:name/certificates
```

Comment:
Should this also have a `GET` method, for `kontena certificate list`, which would show what certs are expiring?